README.md

```diff
@@ -17,19 +17,23 @@
 
 # Cloud infrastructure as data in PostgreSQL
 
-[IaSQL](https://iasql.com) is an open-source SaaS that treats infrastructure as data by maintaining a 2-way connection between a cloud account and a PostgreSQL database.
+[IaSQL](https://iasql.com) is open-source software that treats infrastructure as data by maintaining a 2-way connection between a cloud account and a PostgreSQL database.
+
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./site/static/img/ec2-typewriter_dark.gif">
+  <source media="(prefers-color-scheme: light)" srcset="/site/static/img/ec2-typewriter.gif">
+  <img width="700"/>
+</picture>
 
 ## ⚡️ Try out IaSQL
```

Comment:
will it be worth mentioning possible firewall rules blocking that, or what to do it ports are already in use? (i hit the port in use quite a lot from my mac)

Reply:
https://github.com/iasql/iasql/issues/1950